requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
ran into another road block when I tried removing `mock` from requirements.txt:
```
  File "./corehq/ex-submodules/couchforms/util.py", line 11, in <module>
    from corehq.util.test_utils import unit_testing_only
  File "./corehq/util/test_utils.py", line 6, in <module>
    import mock
ImportError: No module named mock
```
`unit_testing_only` is used on production code to mark functions that are only to be called by tests. I may look into this later...